test/extensions/staged-expenditure.js

```diff
@@ -323,4 +337,227 @@ contract("Staged Expenditure", (accounts) => {
       );
     });
   });
+
+  describe("Interactions with other extensions", async () => {
```

Comment:
Looking at this implementation, I'm inclined to argue that we should put these types of tests in the motions test suite, rather than in the suite associated with the extension. The argument is that we already having a mining process set up in that test suite, so it would be marginally easier to incorporate the test.